src/Microsoft.Health.Dicom.Client/Models/ExtendedQueryTagLevel.cs

```diff
@@ -6,22 +6,22 @@
 namespace Microsoft.Health.Dicom.Client.Models
 {
     /// <summary>
-    /// Level of a custom tag.
+    /// Level of a queryable tag.
     /// </summary>
-    public enum CustomTagLevel
+    public enum ExtendedQueryTagLevel
```

Comment:
Is this `enum` really specific to extended query tags? Does such an `enum` already exist for DICOM itself? Otherwise, perhaps this is just `TagLevel`?

Reply:
This enum should be applied to any DicomTag, so TagLevel could be a better name. 
Unfortunately, there is no existing enum (AFAIK)
will fix it.

Reply:
may be QueryTagLevel is more accurate? Since it  only applies to QIDO